src/JuliusSweetland.OptiKey/Models/KeyValueLink.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.ComponentModel;
+using System.Configuration;
+using System.Globalization;
+using System.Runtime.Serialization;
+using System.Security.Permissions;
+using System.Text;
+using System.Xml;
+using System.Xml.Serialization;
+using JuliusSweetland.OptiKey.Enums;
+using JuliusSweetland.OptiKey.Extensions;
+
+namespace JuliusSweetland.OptiKey.Models
+{
+    [TypeConverter(typeof(KeyValueLinkConverter))]
+    public class KeyValueLink : KeyValue, IEquatable<KeyValueLink>
```

Comment:
Rename to ChangeKeyboardKeyValue.
Could you extend this so that this key type *could* link to a built in keyboard if required, e.g. to "Alpha"